bigframes/ml/llm.py

```diff
@@ -860,3 +867,213 @@ def to_gbq(self, model_name: str, replace: bool = False) -> GeminiTextGenerator:
 
         new_model = self._bqml_model.copy(model_name, replace)
         return new_model.session.read_gbq_model(model_name)
+
+
+@log_adapter.class_logger
+class Claude3TextGenerator(base.BaseEstimator):
+    """Claude3 text generator LLM model.
+
+    .. note::
+
+        This product or feature is subject to the "Pre-GA Offerings Terms" in the General Service Terms section of the
+        Service Specific Terms(https://cloud.google.com/terms/service-terms#1). Pre-GA products and features are available "as is"
+        and might have limited support. For more information, see the launch stage descriptions
+        (https://cloud.google.com/products#product-launch-stages).
+
+    Args:
+        model_name (str, Default to "claude-3-sonnet"):
```

Comment:
[nit] "Defaults to ..."?

Reply:
We use "Default to" in all the APIs... Keeping it for now.